### DISCUSSION TITLE
Allow drag & drop sorting in M2A interface

### DISCUSSION THREAD

**Discussion by lluishi93** [directus/directus#15789] [Archive]
It would be nice to allow some kind of drag & drop sorting for the M2A interface from the APP.

**Comment by benhaynes** [comment]
Should exist! Try adding a sort field to the junction collection.

**Reply by lluishi93** [reply]
Hi @benhaynes , it doesn't let me select one 
![image](https://user-images.githubusercontent.com/14786440/192821466-e800e6bb-40ba-4b29-b4f6-062185fadec9.png)

**Reply by lluishi93** [reply]
Oh, you mean to the JUNCTION collection actually. I'll give a try.

**Reply by benhaynes** [reply]
It's also worth noting that The sort field might not add a column... but if you create a integer field, it should be selectable in that interface. But yes, you want to do it on the junction table.

**Reply by lluishi93** [reply]
Yeah, it works like a charm. Thank you!

**Reply by rijkvanzanten** [reply]
> It's also worth noting that The sort field might not add a column... but if you create a integer field, it should be selectable in that interface. But yes, you want to do it on the junction table.

Also you can type any name in that input and it'll auto-create the field if it doesn't exist yet 🙂
